Frends.AzureBlobStorage.ListBlobsInContainer/Frends.AzureBlobStorage.ListBlobsInContainer/ListBlobsInContainer.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.ComponentModel;
+using Azure.Storage.Blobs;
+using Azure;
+using Frends.AzureBlobStorage.ListBlobsInContainer.Definitions;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs.Models;
+using System.Collections.Generic;
+#pragma warning disable CS1573
+
+namespace Frends.AzureBlobStorage.ListBlobsInContainer
+{
+    public class AzureBlobStorage
+    {
+        /// <summary>
+        ///     List of blobs in container.
+        ///     [Documentation](https://tasks.frends.com/tasks#frends-tasks/Frends.AzureBlobStorage.ListBlobsInContainer)
+        /// </summary>
+        /// <param name="source">Source values.</param>
+        /// <returns>List of blobs in container with chosen listing structure.</returns>
```

Comment:
The return documentation should have information about properties of the Result-object instead of text documentation, so the user is able to see what properties are available in the Result-object and what are their data type. Like `object { string property1, bool property2 etc. }`.